test/fastly-rails_test.rb

```diff
@@ -47,10 +49,27 @@
       assert_equal password, configuration.password
       assert_equal max_age, configuration.max_age
       assert_equal service_id, configuration.service_id
+      assert_equal true, configuration.cache_headers_enabled?
+      assert_equal true, configuration.purging_enabled?
     end
 
     it 'should return a valid client' do
       assert_instance_of FastlyRails::Client, client
     end
   end
+
+  describe 'purge_by_key' do
```

Comment:
It would be best to also test that the method does not execute `client.purge_by_key` when `purging_enabled` is `false`.